phylogenetic/defaults/auspice_config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "Real-time tracking of rabies full genome virus evolution",
+  "maintainers": [
+    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
+  ],
+  "data_provenance": [
+    {
+      "name": "GenBank",
+      "url": "https://www.ncbi.nlm.nih.gov/genbank/"
+    }
+  ],
+  "build_url": "https://github.com/nextstrain/rabies",
+  "colorings": [
+    {
+      "key": "gt",
+      "title": "Genotype",
+      "type": "categorical"
+    },
+    {
+      "key": "region",
+      "title": "Region",
+      "type": "categorical"
+    },
+    {
+      "key": "country",
+      "title": "Country",
+      "type": "categorical"
+    },
+    {
+      "key": "host",
+      "title": "Host",
+      "type": "categorical"
+    }
+  ],
+  "geo_resolutions": [
+    "country",
+    "region"
+  ],
+  "display_defaults": {
+    "map_triplicate": true,
+    "color_by": "region"
+  },
+  "filters": [
+    "region",
+    "country",
+    "author"
+  ],
+  "metadata_columns": [
+    "author"
+  ]
```

Comment:
Flagging that these `metadata_columns` get ignored because you are providing `--metadata-columns` to the export command. 

Consider consolidating them in the `auspice_config.json` or the `config.yaml`.

Reply:
I consolidated these in the `auspice_config.json` in [c49d1c7](https://github.com/nextstrain/rabies/pull/3/commits/c49d1c768bb0c321ca9a2ba5b90fa79ac9834c74).